visidata/experimental/guide.py

```diff
@@ -86,4 +91,15 @@ def iterload(self):
                 yield [i] + list(m.groups())
                 i += 1
 
-BaseSheet.addCommand('', 'open-guide', 'vd.push(GuideGuide("VisiData_Guide"))')
+@VisiData.api
+def getGuide(vd, guidename):
+    if guidename in vd.guides:
+        return vd.guides[guidename]
+    vd.warning(f'there is no guide: {guidename}')
+
+BaseSheet.addCommand('', 'open-guide', 'vd.push(GuideGuide("VisiData_Guide"))', 'opens guide to features in VisiData')
+GuideGuide.addCommand(ENTER, 'open-guide-row', 'vd.push(vd.getGuide(cursorRow[1]))')
```

Comment:
Standard Sheet has `open-row` already which calls `openRow(row)` which is what you would need to override. (This way if someone wants to bind e.g. `Alt+O` to `open-row` it will take effect for all sheets).